stable/jenkins/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: jenkins
 home: https://jenkins.io/
-version: 0.34.0
+version: 0.34.1
```

Comment:
Are we following semantic versioning here?  The change in #11960 introduced a backwards incompatible change that threw us off.

Although this is in fact a patch, I propose we bump the major version here to signal that #11960 was not backwards compatible, and folks should apply with care.

Reply:
we are still in major version 0, breaking changes should be anticipated... wouldn't you agree?

Reply:
@pedrohdz I noticed that as well, however as this was a patch to fix a change which was introduced there already (ie, it was already broken), I didn't see this as really backwards incompatible. I can swap it to a minor version increase if that's preferred.

Reply:
@maorfr @nrmitchi ,
Good points!  Thanks!